wpilibj/src/main/java/edu/wpi/first/wpilibj/Watchdog.java

```diff
@@ -19,13 +19,13 @@
  */
 public class Watchdog implements Closeable, Comparable<Watchdog> {
   // Used for timeout print rate-limiting
-  private static final long kMinPrintPeriod = 1000000; // microseconds
+  private static final long kMinPrintPeriodMicroS = 1000000;
```

Comment:
see above about E notation

Reply:
Since 1e6 is floating point, it causes a warning and build failure